data/human/Republic Paradise Threatened.TXT

```diff
@@ -0,0 +1,301 @@
+			`	Your sigh of relief is short-lived, however, as Agent Norris is shaking his head.`
+			`	"They are trying to stall us," he says. "Probably so that they can force their injunction through. Refuel if you need to, but don't let them drag it out any more than necessary."`
+
+
+mission "Investigate the Source C"
+	autosave
+	landing
+	name "Investigate the Source"
+	description "Track down the source of the plutonium used for the Free Worlds nukes."
+	deadline
+	passengers 1
+	priority
+	source "Placer"
+	destination "Quicksilver"
+	npc
+		personality disables plunders harvests staying
+		government "Pirate"
+		system "Al Dhanab"
+		fleet "Small Core Pirates"
+	npc
+		personality disables plunders harvests staying
+		system "Algol"
+		fleet "Large Core Pirates"
+	to offer
+		has "Investigate the Source B: done"
+	on offer
+		conversation
+			`Agent Norris collects his sample and you make a short hop to the spaceport to refuel. The ground crew does seem a little more disorganized than usual but Agent Norris goes out to bully them into getting their work done as quickly as possible while you take the opportunity to do some quick maintenance. Soon enough <ship> is fully fueled and you seal the hatch for takeoff.
+			`	"Where to this time?" you ask.`
+			`	"<destination>," Norris says simply. He turns to go back to his bunk but just as he does so the same voice as before comes over the channel to <source> control.
+			`	"<ship>, please advised, we have received warnings of increased pirate activity in the region, recommend delaying departure until it has been dealt with."`
+			`	Norris simply gestures for you to take this.
+			choice
+				`	"A bluff?"`
+				`	"What do you think?"`
+				`	"They'll try anything, won't they?"`
+			`	He shrugs. "You know the stakes as well as I do, Captain <last>. I see this as a good sign, they're desparate."`
+			`	You clear your throat and then key the comm. "Thank you <source> control. We will be careful."`
+	on enter "Al Dhanab"
+		dialog
+			`It looks like the warning from <source> control about pirates was real. As you jump emerge from hyperspace you are confronted by a small group who are already plundering.`
+	on enter "Algol"
+		conversation
+			`You emerge in <system> to find a large fleet of pirates waiting. Agent Norris swears, looking flustered for the first time since you met him.`
+			`	"This can't be coincidence!" he growls. "A pirate fleet this large in a small system like <system>? It makes no sense. Somebody has paid them off!"`
+			choice
+				`	"Don't worry, that's why they have me ferrying you around."`
+				`	"They might be strong but <ship> is faster!"`
+				`	"As long as we can make it into the atmosphere I can evade them."`

```

Comment:
```suggestion
				`	"As long as we can make it into the atmosphere, I can evade them."`
```
Haven't you already taken off?

Reply:
This one is (supposed to be) upon entering the destination system, they encounter a blockade but need to make it to the planet.